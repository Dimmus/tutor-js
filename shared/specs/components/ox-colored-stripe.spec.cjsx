React = require 'react'
SnapShot = require 'react-test-renderer'

OXColoredStripe = require '../../src/components/ox-colored-stripe'

describe 'OX Colored Stripe', ->

  it 'renders and matches snapshot', ->
    expect(SnapShot.create(<OXColoredStripe />).toJSON()).toMatchSnapshot()
