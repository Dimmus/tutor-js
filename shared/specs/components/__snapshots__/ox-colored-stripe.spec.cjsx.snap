exports[`OX Colored Stripe renders and matches snapshot 1`] = `
<div
  className="ox-colored-stripe">
  <div
    className="orange" />
  <div
    className="blue" />
  <div
    className="red" />
  <div
    className="yellow" />
  <div
    className="teal" />
</div>
`;
